bounties_api/std_bounties/client_helpers.py

```diff
@@ -172,17 +173,29 @@ def map_user_data(data_hash, public_address):
     plucked_data = pluck(data, user_data_keys)
     plucked_social = pluck(data.get('social', {}), user_social_keys)
 
+    profileDirectoryHash = data.get('profilePhoto', {}).get('fileDirectoryHash', None)
+
     user = {
         **plucked_data,
         **plucked_social,
         'website': data.get('social', {}).get('personalWebsite', None),
         'profileFileName': data.get('profilePhoto', {}).get('fileName', None),
-        'profileDirectoryHash': data.get('profilePhoto', {}).get('fileDirectoryHash', None)
+        'profileDirectoryHash': profileDirectoryHash,
+        'isProfileImageDirty': isProfileImageDirty(profileDirectoryHash, public_address)
     }
 
     return prune(user)
 
 
+def isProfileImageDirty(ipfs_hash, public_address):
```

Comment:
It is definitely weirdly worded. I'm open to better suggestions! I was trying to use similar terminology used in caching system, but it isn't widely used in our codebase

Reply:
Yeah, `dirty` has more understood connotations. To get the same meaning using other words, it would need to both be `modified` but also have the concept of being modified elsewhere. `isProfileImageModifiedUpstream` or something, which is seriously verbose. Going with `dirty` is probably best.